docs/_source/components-overview/pop_syn/population_params.rst

```diff
@@ -377,69 +630,165 @@ These parameters contain options on how the population is evolved, in practical
-     - False (progress bar)
-   * - `breakdown_to_df`
-     - True (convert BinaryStars into DataFrames after evolution)
-   * - `use_MPI`
-     - True ( if True evolve with MPI, equivalent to the following: from mpi4py import MPI, comm = MPI.COMM_WORLD)
-   * - `metallicity`
-     - [2., 1., 0.45, 0.2, 0.1, 0.01, 0.001, 0.0001] (In units of solar metallicity)
-   * - `entropy`
-     - `None` (Random Number Generation: uses system entropy (recommended))
-   * - `number_of_binaries`
-     - 1000000 (int)
-   * - `star_formation`
-     - 'burst' (options are 'constant' 'burst' 'custom_linear' 'custom_log10' 'custom_linear_histogram' 'custom_log10_histogram')
-   * - `max_simulation_time`
-     - 13.8e9 (float in [0,inf])
-   * - `binary_fraction`
-     - 1 (float 0< fraction <=1)
-   * - `primary_mass_scheme`
-     - 'Kroupa2001' (options are 'Salpeter', 'Kroupa1993', 'Kroupa2001')
-   * - `primary_mass_min`
-     - 6.5 (float in [0,300])
-   * - `primary_mass_max`
-     - 250.0 (float in [0,300])
-   * - `secondary_mass_scheme`
-     - 'flat_mass_ratio' (options are 'flat_mass_ratio', 'q=1')
-   * - `secondary_mass_min`
-     - 0.35 (float in [0,300])
-   * - `secondary_mass_max`
-     - 250.0 (float in [0,300])
-   * - `orbital_scheme``
-     - 'period' (options are 'separation', 'period')
-   * - `orbital_period_scheme`
-     - 'Sana+12_period_extended' (used only for orbital_scheme = 'period')
-   * - `orbital_period_min`
-     - 0.75 (float i [0,inf])
-   * - `orbital_period_max`
-     - 6000.0 (float i [0,inf])
-   * - `#orbital_separation_scheme`
-     - 'log_uniform' (used only for orbital_scheme = 'separation', 'log_uniform', 'log_normal')
-   * - `#orbital_separation_min`
-     - 5.0 (float i [0,inf])
-   * - `#orbital_separation_max`
-     - 1e5 (float i [0,inf])
-   * - `#log_orbital_separation_mean`
-     - None (float i [0,inf] used only for orbital_separation_scheme ='log_normal')
-   * - `#log_orbital_separation_sigma`
-     - None (float i [0,inf] used only for orbital_separation_scheme ='log_normal')
-   * - `eccentricity_scheme`
-     - 'zero' (options are 'zero', 'thermal', 'uniform')
+  * - Parameter
+    - Description
+    - Default Value
+
+  * - ``optimize_ram``
+    - | Save population in batches.
+    - ``True``
+
+  * - ``ram_per_cpu``
+    - | Set maximum RAM per CPU before batch saving (in GB).
+    - ``None``
+
+  * - ``dump_rate``
+    - | Batch save after evolving N binaries.
+    - ``2000``
+
+  * - ``temp_directory``
+    - | Folder for keeping batch files.
+    - ``'batches'``
+
+  * - ``tqdm``
+    - | Enable progress bar.
+    - ``False``
+
+  * - ``breakdown_to_df``
+    - | Convert BinaryStars into DataFrames after evolution.
+    - ``True``
+
+  * - ``use_MPI``
+    - | If True, evolve with MPI (equivalent to: ``from mpi4py import MPI, comm = MPI.COMM_WORLD``).
+    - ``True``
+
+  * - ``metallicity``
+    - | In units of solar metallicity.
+    - ``[2., 1., 0.45, 0.2, 0.1, 0.01, 0.001, 0.0001]``
```

Comment:
You have changed the default back in the [population_params_default.ini](https://github.com/POSYDON-code/POSYDON/blob/max_tutorial_updates/posydon/popsyn/population_params_default.ini), but not here. The same applies to the number of binaries. Please, make it consistent.
May just thinking a bit further: Is there a way to read those default values automatically from the population_params_default.ini? So, we would only update one place in the future, and the docs get autoupdated accordingly.